src/rabbit_control_main.erl

```diff
@@ -519,6 +519,9 @@ action(list_vhosts, Node, Args, _Opts, Inform, Timeout) ->
     display_info_list(call(Node, {rabbit_vhost, info_all, []}, Timeout),
                       ArgAtoms);
 
+action(list_user_permissions, _Node, _Args = [], _Opts, _Inform, _Timeout) ->
```

Comment:
Perhaps `Args = [""]` could be caught here as well?

I'm thinking of scripts calling `rabbitmqctl list_user_permissions "$user"` but `$user` is not set.